L1Trigger/L1THGCal/src/concentrator/HGCalConcentratorSuperTriggerCellImpl.cc

```diff
@@ -13,9 +13,9 @@ HGCalConcentratorSuperTriggerCellImpl::getSuperTriggerCellId(int detid) const {
     return TC_id.cell(); //scintillator
   } else {
     int TC_wafer = TC_id.wafer();
-    int split = 0x3a;
+    static const int split = 0x3a;
```

Comment:
The comment was to make these variables class members (defined in the class definition, not inside the function)